### PR TITLE
Add fluentbit setup

### PR DESCRIPTION
**Pull request by shuse2**
### Description

Add fluentbit setup to send the logs to firehose

### CODE REVIEW

File: fluentbit_config/parsers.conf [hunk 1]
```diff
@@ -0,0 +1,6 @@
+# parsers.conf
+[PARSER]
+    Name        json
+    Format      json
+    Time_Key    time
+    Time_Format %Y-%m-%dT%H:%M:%SZ
```
Comment (sameersubudhi) [inline fluentbit_config/parsers.conf:6]: Is the `Z` suffix intentional, i.e. for UTC?

**Review by sameersubudhi — APPROVED** [review]
Looks good and works as expected! 👍 
Please consider the following changes though.

File: docker-compose.yml [hunk 2]
```diff
@@ -45,19 +43,34 @@ services:
...
-      options:
-        awslogs-region: eu-west-3
-        awslogs-group: /ec2/lisk-across-relayer
-        awslogs-stream: ec2/lisk.across.rebalancer.docker.log
+      driver: fluentd
     environment:
       - REDIS_URL=redis://redis:6379
 
+  fluent-bit:
+    image: amazon/aws-for-fluent-bit:2.32.2
```
[the remark this answers is not in the document]
Comment (shuse2) [reply docker-compose.yml:58]: the amazon github says

> Version number tag: Each release has a version number, for example 2.28.4. These are the only tags we recommend ✅😍 consuming in production environments: [Guidance on consuming versions](https://github.com/aws/aws-for-fluent-bit#guidance-on-consuming-versions).

File: docker-compose.yml
[the remark this answers is not in the document]
Comment (shuse2) [reply docker-compose.yml:1]: updated

File: fluentbit_config/parsers.conf (hunk 1 above)
Comment (shuse2) [reply fluentbit_config/parsers.conf]: the parsers.conf was not used anymore, so removed

**Review by matjazv — APPROVED** [review]

**Pull request merged**